cesium/db.go

```diff
@@ -45,11 +46,15 @@ type DB struct {
 		inlet  confluence.Inlet[WriterRequest]
 		outlet confluence.Outlet[WriterResponse]
 	}
+	closed   bool
 	shutdown io.Closer
 }
 
 // Write implements DB.
 func (db *DB) Write(ctx context.Context, start telem.TimeStamp, frame Frame) error {
```

Comment:
Can we get better documentation on these methods. These no longer 'implement DB' because DB is the concrete type itself. Same with WriteArray and Read